apps/web-api/src/tokens/token.dto.ts

```diff
@@ -2,6 +2,16 @@ import { Tokens } from '@entities/Tokens';
 import { Field, Int, ObjectType } from '@nestjs/graphql';
 import { GraphQLJSONObject } from 'graphql-type-json';
 
+export enum TokenEnum {
```

Comment:
I don't like name TokenEnum. Maybe should choose name that reflects its purpose more clearly?